src/Discord/Internal/Rest/Channel.hs

```diff
@@ -102,18 +101,18 @@ data ChannelRequest a where
 data MessageDetailedOpts = MessageDetailedOpts
   { messageDetailedContent                  :: T.Text
   , messageDetailedTTS                      :: Bool
-  , messageDetailedEmbed                    :: Maybe CreateEmbed
+  , messageDetailedEmbeds                    :: Maybe [CreateEmbed]
   , messageDetailedFile                     :: Maybe (T.Text, B.ByteString)
   , messageDetailedAllowedMentions          :: Maybe AllowedMentions
   , messageDetailedReference                :: Maybe MessageReference
-  , messageDetailedComponents               :: Maybe [Component]
+  , messageDetailedComponents               :: Maybe [ComponentActionRow]
   , messageDetailedStickerIds               :: Maybe [StickerId]
-  } deriving (Show, Read, Eq, Ord)
+  } deriving (Show, Eq, Ord)
```

Comment:
is there a reason `MessageDetailedOpts` can't be `Read` anymore? if so that's okay